doc/api/vm.md

```diff
@@ -417,8 +408,9 @@ const contextifiedSandbox = vm.createContext({ secret: 42 });
 
 * `code` {string} JavaScript Module code to parse
 * `options`
-  * `url` {string} URL used in module resolution and stack traces. **Default:**
-    `'vm:module(i)'` where `i` is a context-specific ascending index.
+  * `specifier` {string} String used in module resolution and stack traces.
```

Comment:
Usually `specifier` is used to refer to the (possibly) unresolved module specifier in an import statement, as opposed to the resolved id of a module. `url` made sense to me as the user access of that is via `import.meta.url` which we may want to support by default in future.

If we must change from `url` to indicate that it is _not necessarily_ a URL, perhaps lets call it `id` instead?

Reply:
I'd be fine with `identifier`, although I do prefer `specifier` :P

Reply:
`identifier` could work!